### ISSUE TITLE
Focus indicator on New and Edit buttons is hard to see

### ISSUE THREAD

**Issue by meganvilla** [OpenSpace/OpenSpace#3091]
A focus indicator is present on the "New" and "Edit" buttons, but the gray (#9F9F9F) 1 px dotted outline is extremely difficult to see

The focus indicator is missing on the “combo box” dropdowns and the "Start" button. Meanwhile, a focus indicator is present on the "New" and "Edit" buttons, but the gray (#9F9F9F) 1 px dotted outline is extremely difficult to see. The resulting color contrast ratio is 2.38:1 against the gray (#606060) background for buttons, and 2.65:1 against their white (#FFF) border. 

**Suggested Solution:** Review the current styling and use a prominent focus indicator via QPushButton:focus, such as a solid outline around the control that meets the minimum non-text contrast ratio requirement of 3:1 with all adjacent colors. Here are some additional considerations: 
Changes in font/background colors and box shadows may be used, as long as they are accompanied by an outline to ensure that there is at least one indication of the focus state that persists in Windows Contrast Themes. Additionally, if the font/background color of the control is changed when focused, the text must still have a color contrast ratio of 4.5:1 with its background for normal sized text, and 3:1 for medium/large font size (14 pt and bold or 18 pt).

While consistency is encouraged, the styling of the focus indicator can be adjusted across components so that it remains highly visible and has sufficient color contrast, since different backgrounds may be used in different circumstances.

Resources:
A guide to designing accessible, WCAG-compliant focus indicators: https://www.sarasoueidan.com/blog/focus-indicators/  
Success Criterion 1.4.11: Non-text Contrast: https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html  
Success Criterion 2.4.11 Focus Appearance: https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance-minimum 

Severity level 1 from PAC
